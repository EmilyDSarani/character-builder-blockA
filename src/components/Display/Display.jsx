import React from 'react'

export default function Display({catchphrase}) {
    console.log(catchphrase)
    return (
        <div>
            {catchphrase.map((phrase) => (
                <p key={phrase}>{phrase}</p>
            ))}
        </div>
    );
}
