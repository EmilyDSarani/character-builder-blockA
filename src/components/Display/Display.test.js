import {screen, render} from '@testing-library/react';
import Display from './Display'

it('should render the Display', () => {
    const {container} = render (
    <Display catchphrase={['holy hotwheels batman', 'let it goooooo', 'the lion sleeps toniiiight']} />);

    expect(container).toMatchSnapshot();
});