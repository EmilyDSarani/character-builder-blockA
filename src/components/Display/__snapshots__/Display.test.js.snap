// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render the Display 1`] = `
<div>
  <div>
    <p>
      holy hotwheels batman
    </p>
    <p>
      let it goooooo
    </p>
    <p>
      the lion sleeps toniiiight
    </p>
  </div>
</div>
`;
